Examples/TinyTableRest/index.html

```diff
@@ -1,63 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Please restore :)

Reply:
Um, I think that was a .gitignore entry put in by a certain miklosduma